service.go

```diff
@@ -40,7 +40,7 @@ type (
 		Port                      int    `json:"port"`
 		Route                     string `json:"route"`
 		Usage                     string `json:"usage"`
-		Ssl                       string `json:"ssl"`
+		Ssl                       bool   `json:"ssl"`
```

Comment:
Should be `*bool` so that we can distinguish between value not present and false?